### DISCUSSION TITLE
Unsupported mcs table 4

### DISCUSSION THREAD

**Discussion by pustovale** [NVlabs/sionna#146] [Q&A] [answered]
Dear developers,

I have a question what is the reason  Sionna does not support mcs table 4.
There is assert in tb_config.py

```
def mcs_table(self, value):
        assert value in range(1,4), 
            "mcs_table must be in range from 1 to 3"
```

After disabling this assert I was able to set mcs table 4 and run few mcs from there. But it is not clear for me why the assert was set in the first place.

**Comment by SebastianCa** [comment, marked as answer]
Hi,

you are right, the MCS table 4 configuration should be allowed in the `TBConfig`.

Thanks for raising this issue. Will be fixed in the next release.
